content/ios/user.md

```diff
@@ -79,11 +91,15 @@ if (container.currentUserRecordID) {
 
 ## Logging out
 
+Of course, you also would want to allow users to log out of their accounts by using `logoutWithCompletionHandler:`.
```

Comment:
Words like "Of course" should not appear in the documentation unless for special cases.

Better keep the sentence short and simple, I think "You can log out the current user by `logoutWithCompletionHandler`" is sufficient.